src/test/frontend/test/node_helper.cljs

```diff
@@ -0,0 +1,17 @@
+(ns frontend.test.node-helper
+  "Common helper fns for node tests"
+  (:require ["path" :as node-path]
+            ["fs" :as fs-node]))
+
+(defn create-tmp-dir
```

Comment:
Have to move this fn to this separate namespace to run the tests.

Reply:
@tiensonqin I added https://github.com/logseq/logseq/commit/8fca36629f9072261143e3b5a00d9916d01f7e53. Feel free to fix/enhance any of it